edx_solutions_api_integration/users/views.py

```diff
@@ -1253,11 +1254,14 @@ class UsersOrganizationsList(SecureListAPIView):
     serializer_class = BasicOrganizationSerializer
 
     def get_queryset(self):
-        user_id = self.kwargs['user_id']
-        try:
-            user = User.objects.get(id=user_id)
-        except ObjectDoesNotExist:
-            raise Http404
+        user_id = self.kwargs.get('user_id', None)
```

Comment:
Can we create a method of extracting user from user_id or username? That will be called in all APIs where we need to do this instead of duplicated the code.

Reply:
Yeah thats a good idea. Moved user retrieval from request params into a helper method.